### DISCUSSION TITLE
Welcome to osmosis Discussions!

### DISCUSSION THREAD

**Discussion by sunnya97** [osmosis-labs/osmosis#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by James-Osmo** [comment]
Hey all, I am so excited to see where this project goes and if it truly does change the way we interact with each other on a global scale (not just financially). I 100 percent believe in this team and have been with Cosmos for awhile now (sadly with Coinbase at first. But, after a bit of research I realize a private wallet is better which is what I have now and have all my cosmos staked).
... there is a new world approaching; So, I am extremely appreciate everyones efforts and the way you all are approaching it. Lets keep this going, it's only the beginning.

**Comment by Sephix2005** [comment]
Same as above, had every single Atom staked on coinbase at first, binance, and now that i realised it's better, they're all staked on cosmostation ! 
Missed the airdrop apparently, but i'm still in, and I really believe in it as well.
Not even scared of drop&rise cycle

Keep it going !

**Comment by LL-Cosmo** [comment]
Looking good. Chatted with Valordragon and got answers to my questions.

**Comment by artifactstaking** [comment]
Hello, I'm looking to run a validator for Osmosis. It looks like you have reached the 100 validator limit, if I stake more will my validator knock the smallest validator out and join the group?
